### PR TITLE
Quad 134 brand

### PR DESCRIPTION
**Pull request by afromme**
This adds the arXiv brand page, brand guide PDF, and brand usage PDF.

### CODE REVIEW

File: about/brand.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+# arXiv's Brand and Identity
+
+arXiv is a place of connection, linking together people and ideas, and connecting them with the world of open science. 
+
+We aim to cultivate an identity that reflects arXiv’s true nature as a professional, innovative tool created by and for researchers. arXiv’s brand guide and usage guidelines ensure that we express arXiv's identity with consistent quality and continuity across communication channels. Please consult the following documents for more information.
```
Comment (SBBCornell) [inline about/brand.md:6]:
```suggestion
We aim to cultivate an identity that reflects arXiv’s true nature as a professional, innovative tool created by and for researchers. arXiv’s brand guide and usage guidelines ensure that we express arXiv's identity with consistent quality and continuity across communication channels, and protect it from unauthorized usage that causes confusion among our community of users. Please consult the following documents for more information. 
```

**Review by SBBCornell — APPROVED** [review]
Looks great! I just added a possible suggestion to include a note about protection too.

File: about/brand.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+arXiv is a place of connection, linking together people and ideas, and connecting them with the world of open science. 
+
+We aim to cultivate an identity that reflects arXiv’s true nature as a professional, innovative tool created by and for researchers. arXiv’s brand guide and usage guidelines ensure that we express arXiv's identity with consistent quality and continuity across communication channels, and protect it from unauthorized usage that causes confusion among our community of users. Please consult the following documents for more information. 
+
+* [arXiv Brand Guide](/about/arXiv_Brand_Guide.pdf)
+
+* [arXiv Brand Usage Guidelines](/about/arXiv_Brand_Usage.pdf)
+
+*Note: Use of the names “arXiv,” “arXiv.org,” “arXiv Labs,” “arXivLabs,” and associated logos and colors are only allowed with the explicit written permission from the arXiv management team. Questions? Email Nextgen@arXiv.org.*
```
Comment (jimentwood) [inline about/brand.md:12]:
```suggestion
*Note: Use of the names “arXiv,” “arXiv.org,” “arXiv Labs,” “arXivLabs,” and associated logos and colors are only allowed with the explicit written permission from the arXiv management team. Questions? Email nextgen@arXiv.org.* 
```

Looks like this is typically lower case in other uses nextgen@arXiv.org

**Comment by jimentwood** [comment]
I wonder if we should embed a date or version into the PDF filenames. 
Ex: arXiv_Brand_Guide_v2102.pdf
Maybe not needed. I'm trying to envision people using this as the guide evolves in the future.

**Review by jimentwood — APPROVED** [review]
Looks good. Comment on filenames totally optional.

**Comment by afromme** [comment]
> I wonder if we should embed a date or version into the PDF filenames.
> Ex: arXiv_Brand_Guide_v2102.pdf
> Maybe not needed. I'm trying to envision people using this as the guide evolves in the future.

@jimentwood done!

**Pull request merged**